english/movie-ripoffs/zoolander/zoolander.md

```diff
@@ -12,9 +14,11 @@ his eyes unsuccessfully attempting to focus on you as he inhales deeply from a h
 Looking puzzled Hansel continues. "Dunno. I was climbing Mt Vesuvius and I slipped and fell. Next thing I was here."
 He pauses for a moment "Oh... epiphany... I've never even BEEN to Vesuvius!  Must be something to do with the pills." 
 
-Excitedly, Hansel rummages through his pockets and hauls out a grubby assortment of colored tablets.  
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.